packages/core/integration-tests/test/integration/formats/esm-async-commonjs/index.js

```diff
@@ -0,0 +1 @@
+output = import('./async').then(a => a.foo + 2);
```

Comment:
How would you access `module.exports` here? `a` should be the string `"async"` right? Looks like with the logic below you'd actually have to do `a.default` though...

Reply:
I guess that's not really possible with ESM though...